README.md

```diff
@@ -80,33 +80,72 @@ metric exporters, service monitors, and custom dashboards.
 
 ```text
-- user.env
-- [monitoring]
-  - user.env
-  - user-values-prom-operator.yaml
-  - user-values-pushgateway.yaml
-- [logging]
-  - user.env
-  - user-values-elasticsearch-open.yaml
-  - user-values-es-exporter.yaml
-  - user-values-fluent-bit-open.yaml
+/my-viya4mon-user-dir/user.env
+
+/my-viya4mon-user-dir/monitoring/user.env
+/my-viya4mon-user-dir/monitoring/user-values-prom-operator.yaml
+/my-viya4mon-user-dir/monitoring/user-values-pushgateway.yaml
+
+/my-viya4mon-user-dir/logging/user.env
+/my-viya4mon-user-dir/logging/user-values-elasticsearch-open.yaml
+/my-viya4mon-user-dir/logging/user-values-es-exporter.yaml
+/my-viya4mon-user-dir/logging/user-values-fluent-bit-open.yaml
 ```
 
+You specify the environment variables in the `user.env` files and the Helm chart parameters in the `*.yaml` configuration files. 
+
+You do not have to specify values in all of these files. The deployment process ignores any files in the `USER_DIR` other than these.
+
+#### Specifying Environment Variables in user.env Files
+
+Environment variables control script behavior and high-level options such as TLS and workload node placement. Note that you can also specify environment variables on a command line, but specifying the variables in `user.env` is recommended, in order to maintain a consistent set of values for future deployments. The values in the top-level `user.env` file (`/my-viya4mon-user-dir/user.env`) apply to both the monitoring and logging deployments. The values in `/my-viya4mon-user-dir/monitoring/user.env` apply only to the monitoring deployment and the values in `/my-viya4mon-user-dir/logging/user.env` apply only to the logging deployment.
+
+#### Using Sample Configuration Files
+
+This repository contains several sample configuration files which provide values for specific situations. The `generic-base` sample contains a complete set of customization files and comments in each of the files to enable you to understand the how the values are used. A best practice is to use the files in the `generic-base` sample as a base for your customization files, and then change the values or copy values from other samples to match your situation.  
+
+ Alternatively, you can copy the configuration files for the sample that most closely matches your environment from the repository to your customization file directory. This enables you to start your customization with a set of values that are valid for your situation. You can then make further modifications to the files. 
+
+If more than one sample applies to your environment, you can manually copy the values from the other sample files to the files in your customization directory.
+
+See the [monitoring README](monitoring/README.md) and [logging README](logging/README.md) for information about determining valid values for the `user.env` and `*.yaml` files.
```

Comment:
I don't think there is much documentation on the values in `user.env` outside of the sample `user.env` files.